src/yashCJun20/Assignment19/StringArray.java

```diff
@@ -0,0 +1,27 @@
+package yashCJun20.Assignment19;
+
+public class StringArray {
+	
+	void toFindAllDifferIndex (String [] arr1,String [] arr2){
+		
+		for (int i=0;i<arr1.length;i++) {
+			
+        	if (arr1[i]!=arr2[i]) {
```

Comment:
You should use .equals() method for String comparison.

Reply:
As advised,changes made to the code.
Please review.
I will take care of this for the upcoming assignments